caravel/controllers/moderation.py

```diff
@@ -4,11 +4,27 @@
 from caravel.model import moderation
 from flask import render_template, url_for, request
 import user_agents
+import itertools
+import re
 
 from google.appengine.ext import ndb
 from google.appengine.api import users
 
 
+class ModerationConfig(ndb.Model):
+    """
+    Defines the static configuration of the automoderator.
+    """
+
+    @classmethod
+    def get(klass):
```

Comment:
In this case, there's only ever supposed to be one ModerationConfig — so this step allows us to get that (singleton) entity.